wandb/sdk/launch/utils.py

```diff
@@ -96,10 +97,12 @@ def construct_launch_spec(
     # override base config (if supplied) with supplied args
     launch_spec = launch_config if launch_config is not None else {}
     launch_spec["uri"] = uri
-    if entity:
-        launch_spec["entity"] = entity
-    if project:
-        launch_spec["project"] = project
+    backup_project, backup_entity = set_project_entity_defaults(
```

Comment:
hmm nitpicky but can we either pass in the user supplied `target` and `entity` into this fn and handle the priority there (preferable i think) or handle the user supplied override before this fn is called. would like to preserve the contract of like "after this gets called you know what project and entity you're using"

Reply:
Fixed